pro/src/components/pages/Offerers/Offerer/VenueV1/fields/BankInformationFields/BankInformationFieldsWithBusinessUnit.jsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * @debt complexity "Gaël: file nested too deep in directory structure"
+ * @debt directory "Gaël: this file should be migrated within the new directory structure"
+ */
+
+import PropTypes from 'prop-types'
+import React, { useEffect, useState } from 'react'
+import { Field } from 'react-final-form'
+
+import Banner from 'components/layout/Banner/Banner'
+import Icon from 'components/layout/Icon'
+import { getBusinessUnits } from 'repository/pcapi/pcapi'
+import { DEMARCHES_SIMPLIFIEES_VENUE_RIB_UPLOAD_PROCEDURE_URL } from 'utils/config'
+
+import { formatSiret } from '../../siret/formatSiret'
+
+import { ApplicationBanner } from './ApplicationBanner'
+
+const BankInformationWithBusinessUnit = ({ readOnly, offerer, venue }) => {
+  const [businessUnits, setBusinessUnits] = useState([])
+  const venueHasBankInformation = !!(venue.iban && venue.bic)
+  const venueHasApplicationId = !!venue.demarchesSimplifieesApplicationId
+  const venueBusinessUnit = businessUnits.filter(
+    businessUnit => businessUnit.id == venue.businessUnitId
+  )
+  const displayVenueBusinessUnit = venueBusinessUnit[0]
+    ? `${formatSiret(venueBusinessUnit[0].siret)} - ${
+        venueBusinessUnit[0].iban
+      }`
+    : ''
+
+  useEffect(() => {
+    getBusinessUnits(offerer.id).then(loadedBusinessUnits => {
+      setBusinessUnits(loadedBusinessUnits)
+    })
+  }, [setBusinessUnits, offerer.id])
+
+  return (
+    <div className="section vp-content-section bank-information">
+      <div className="main-list-title title-actions-container">
+        <h2 className="main-list-title-text">
+          Coordonnées bancaires du lieu (remboursement)
+        </h2>
+        {venueHasBankInformation && (
+          <a
+            className="tertiary-link"
+            href={DEMARCHES_SIMPLIFIEES_VENUE_RIB_UPLOAD_PROCEDURE_URL}
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            <Icon alt="" svg="ico-external-site" />
+            Modifier
+          </a>
+        )}
+      </div>
+      <p className="bi-subtitle">
+        Ces coordonnées bancaires seront utilisées pour les remboursements des
+        offres éligibles de ce lieu.
+      </p>
+      {businessUnits.length && (
+        <div className="field field-select is-label-aligned">
+          <div className="field-label">
+            <label htmlFor="venue-business-unit">
+              Coordonnées bancaires pour vos remboursements :
+            </label>
+          </div>
+          {readOnly ? (
+            displayVenueBusinessUnit
+          ) : (
+            <div className="control control-select">
+              <div className="select">
+                <Field
+                  component="select"
+                  id="venue-business-unit"
+                  name="venueBusinessUnit"
```

Comment:
```suggestion
                  name="businessUnitId"
```
Pourquoi pas `businessUnitId`, ça permet à react-final-form de récupérer la valeur initial à partir de `venue`, en plus côté back en attend `businessUnitId` dans le body